tests/CommunityToolkit.Aspire.Hosting.ActiveMQ.Tests/AppHostTests.cs

```diff
@@ -0,0 +1,50 @@
+using Aspire.Components.Common.Tests;
+using CommunityToolkit.Aspire.Hosting.ActiveMQ.MassTransit;
+using CommunityToolkit.Aspire.Testing;
+using FluentAssertions;
+using System.Net.Http.Json;
+
+namespace CommunityToolkit.Aspire.Hosting.ActiveMQ.Tests;
+
+[RequiresDocker]
+public class AppHostTests(AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_ActiveMQ_MassTransit> fixture) : IClassFixture<AspireIntegrationTestFixture<Projects.CommunityToolkit_Aspire_Hosting_ActiveMQ_MassTransit>>
+{
+    [Fact]
+    public async Task ResourceStartsAndRespondsOk()
+    {
+        const string resourceName = "masstransitExample";
+        await fixture.ResourceNotificationService.WaitForResourceHealthyAsync(resourceName).WaitAsync(TimeSpan.FromMinutes(2));
+        HttpClient httpClient = fixture.CreateHttpClient(resourceName);
+        
+        HttpResponseMessage response = await httpClient.PostAsync("/send/Hello%20World", new StringContent(""));
+
+        response.StatusCode.Should().Be(HttpStatusCode.OK);
+    }
+
+    [Fact]
+    public async Task WhenMessageIsSendItShouldBeReceivedByConsumer()
+    {
+        const string resourceName = "masstransitExample";
+        await fixture.ResourceNotificationService.WaitForResourceHealthyAsync(resourceName).WaitAsync(TimeSpan.FromMinutes(2));
+        HttpClient httpClient = fixture.CreateHttpClient(resourceName);
+        
+        MessageCounter? oldMessageCounter = await httpClient.GetFromJsonAsync<MessageCounter>("/received");
+        oldMessageCounter.Should().NotBeNull();
+        
+        await httpClient.PostAsync("/send/Hello%20World", new StringContent(""));
+
+        MessageCounter? messageCounter = null;
+        for (int i = 0; i < 10; i++)
+        {
+            messageCounter = await httpClient.GetFromJsonAsync<MessageCounter>("/received");
+            await Task.Delay(1000);
```

Comment:
Is this to simulate a delay that might happen between messages going to the endpoint? I'm somewhat bothered by having `Task.Delay` in a test as it results in a forced duration of a test.